### ISSUE TITLE
Better alt text for badges

### ISSUE THREAD

**Issue by Myoldmopar** [lefticus/decent_ci#14]
For whatever reason, sometimes the badges on our Github comments don't render, as such:

![image](https://user-images.githubusercontent.com/849824/29273977-cd3d9214-80cb-11e7-9dd1-12e931d79fc8.png)

It would be great if we could at least include the badge text in the alt text so we could still read the results.

Should be able to just change the line [here](https://github.com/lefticus/decent_ci/blob/16e0a5f6a0f75fd6490158137de2f840dbda11e5/lib/potentialbuild.rb#L1053) and related other places to put the %s text into the badge alt text as well as the badge itself.
